src/platform_impl/windows/dark_mode.rs

```diff
@@ -32,9 +35,9 @@ lazy_static! {
                     szCSDVersion: [0; 128],
                 };
 
-                let status = (rtl_get_version)(&mut vi as _);
+                let status = (rtl_get_version)(&mut vi);
 
-                if NT_SUCCESS(status) && vi.dwMajorVersion == 10 && vi.dwMinorVersion == 0 {
+                if status == S_OK && vi.dwMajorVersion == 10 && vi.dwMinorVersion == 0 {
```

Comment:
Semantic change (`status` is an `NTSTATUS`, not sure how it is supposed to be handled?)

Reply:
NT_SUCCESS seems to be defined as:
```cpp
#define NT_SUCCESS(Status) ((NTSTATUS)(Status) >= 0)
```
according to: https://docs.microsoft.com/en-us/windows/win32/api/bcrypt/nf-bcrypt-bcryptenumcontexts

Reply:
Will change it to >= 0